### ISSUE TITLE
Accepting blank field 

### ISSUE THREAD

**Issue by sadathshariff** [thedevyansh/random_number#18]
Hi @thedevyansh  the input field is accepting the blank field and displaying as zero, this should not happen and it should tell user to enter number can i work on this issue

**Comment by thedevyansh** [comment]
Yes, that's a good suggestion. Please go for it :) I am assigning this issue to you.
